_resources/port1.0/group/stack_check_fix-1.0.tcl

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+#
+# This PortGroup attempts to work around a bug in Xcode 11 on Catalina.
+# See https://forums.developer.apple.com/thread/121887
+#
+# Usage:
+#
+#   PortGroup        stack_check_fix 1.0
+#
+#   stack_check.fix_type:
+#      append_to_compler_flags (add -fno-stack-check to compiler flags)
+#      append_to_compler_name  (add -fno-stack-check to compiler name)
```

Comment:
Is `compler` a typo of `compiler`?

Reply:
Yes, thank you for noticing.
This should be fixed now.